pyconafrica/settings.py

```diff
@@ -354,4 +358,7 @@
     "CLOUD_NAME": os.getenv("CLOUDINARY_CLOUD_NAME"),
     "API_KEY": os.getenv("CLOUDINARY_API_KEY"),
     "API_SECRET": os.getenv("CLOUDINARY_API_SECRET"),
-}
\ No newline at end of file
+}
+
+if DEBUG:
+    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
```

Comment:
since we are to have emailing functionality, does the `EMAIL_BACKEND` require to active when `DEBUG` is `True`?
assuming `DEBUG` will be disabled in production.